### DISCUSSION TITLE
Hardware requirements to setup Spline Server

### DISCUSSION THREAD

**Discussion by talhagithub02** [AbsaOSS/spline#1305] [Q&A] [answered]
Do we have a place where we can see hardware specs needed to run Spline Server or some one can share exact specs?
Your input is appreciated.

**Comment by wajda** [comment, marked as answer]
The REST server is a Java application, so at least 1Gb memory is recommended, better 2Gb.
For ArangoDB the more the better. I would say 8+ per node or even more depending on your load and complexity of the execution plans that you capture. 
Number of CPUs isn't that important - it affects performance especially in case of parallel load, but the amount of RAM is more critical I would say.
